stories/webapp/hierarchy/HierarchyFolderRow.js

```diff
@@ -49,14 +48,7 @@ function HierarchyFolderRow({ data, nameForm, children, open }) {
       style={{ minWidth: '320px' }}
       open={open}
       showToggle
-      onClick={() => {
-        if (!isDisabled) {
-          return isLevelSelected
-            ? deselectMultiple([data.id, ...childIds], 'folder')
-            : selectMultiple([data.id, ...childIds], 'folder');
-        }
-        return null;
-      }}
+      onClick={handleClick}
```

Comment:
I feel like handleClick is a bit too vague of a name, just calling it along the lines of updateSelected would be better

Reply:
👍